packages/terra-site/themeable-variables.json

```diff
@@ -2,16 +2,16 @@
   "Alert": {
     "--terra-alert-icon-font-size": "1.286em",
     "--terra-alert-custom-text-color": "#1c1f21",
-    "--terra-alert-alert-background-color": "#fed1d1",
-    "--terra-alert-error-background-color": "#fed1d1",
+    "--terra-alert-alert-background-color": "#fedcdc",
+    "--terra-alert-error-background-color": "#fedcdc",
```

Comment:
I'm assuming the changes in this file outside of what's underneath Radio were pulled in from master?

Reply:
Actually, I locally removed a boolean check specified in #1106 to get these to update, otherwise the theming plugin wasn't updating these values. We should probably look more into this/remove it in a separate PR.